src/Kernel-Tests/CompiledBlockTest.class.st

```diff
@@ -13,9 +13,9 @@ CompiledBlockTest >> testLiteralEqual [
 
 	| value |
 	value := 0.
-	10 timesRepeat: [self assert: value equals: 0].
+	(1 to: 10) do: [self assert: value equals: 0].
 	value := 1.
-	10 timesRepeat: [self assert: value equals: 0].'.
+	(1 to: 10) do: [self assert: value equals: 0].'.
```

Comment:
Why is `timesRepeat:` replaced by an `interval do:`(that could also have been a `to:do:`)?

Reply:
the test needs a block. the PR changes timesRepeat: to not compile the argument as a block.

As the code is not executed, but is executable in principle I wanted a minimal change...

Reply:
to:do: is compiled optimzid, too, so it would have the same problem as timesRepeat:

Reply:
ok, thanks